Library/Homebrew/service.rb

```diff
@@ -178,10 +173,9 @@ def keep_alive(value = nil)
 
     sig { params(value: T.nilable(T::Boolean)).returns(T::Boolean) }
     def require_root(value = nil)
-      case value
-      when nil
+      if value.nil?
```

Comment:
For boolean barams, we have to explicitly check `.nil?`, which inverts the logic. I could use this same approach for non-boolean param methods for consistency, if desirable.

Reply:
I think it's fine like this, thanks!